config/locales/en/shared.yml

```diff
@@ -440,9 +440,11 @@ en:
         first_name: First name
         last_name: Last name
         changed_last_name: Has your client ever changed their last name?
-        last_name_at_birth: What was your client's last name at birth?
+        last_name_at_birth: Last name at birth
+        last_name_at_birth_question: What was your client's last name at birth?
+        same_as_last_name: Same as last name
         nino: National Insurance number
-        has_partner: Does your client have a partner?
+        has_partner: Client has a partner
```

Comment:
Should this lose the question mark?  442 still has one

Reply:
Good question. It doesn't on the design but you're right that it seems a bit inconsistent so I'll double check

Reply:
I've updated to add a question mark